README.md

```diff
@@ -76,10 +76,25 @@ config.vm.synced_folder ".", "/home/core/share", id: "core", :nfs => true,  :mou
 
 After a 'vagrant reload' you will be prompted for your local machine password.
 
-#### Provisioning with user-data
+#### Provisioning with Ignition (VirtualBox Provider (default))
 
-The Vagrantfile will provision your CoreOS VM(s) with [coreos-cloudinit][coreos-cloudinit] if a `user-data` file is found in the project directory.
-coreos-cloudinit simplifies the provisioning process through the use of a script or cloud-config document.
+When using the VirtualBox provider for vagrant (the default), Ignition is used to provision the machine. This uses a special plugin that is
```

Comment:
If it's the default why do I have to type `--provider=virtualbox`, else have it use libvirt.
This wording seems confusing.

Do we mean 'preferred' or 'recommended' option?